firefox-ios/firefox-ios-tests/Tests/XCUITests/FakespotTests.swift

```diff
@@ -145,7 +145,6 @@ class FakespotTests: IphoneOnlyTestCase {
     }
 
     // https://testrail.stage.mozaws.net/index.php?/cases/view/2358892
-    // Smoketest
```

Comment:
Sorry, I saw an earlier ticket that this test should not be in the smoke test. This comment should be removed earlier.